pkg/signals/signals_posix.go

```diff
@@ -0,0 +1,24 @@
+//go:build !windows
+// +build !windows
+
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package signals
+
+import (
+	"os"
+	"syscall"
+)
+
+var shutdownSignals = []os.Signal{os.Interrupt, syscall.SIGTERM}
```

Comment:
qq: why the other signals are not considered as shutdown?

syscall.SIGHUP, syscall.SIGINT, syscall.SIGQUIT

Reply:
`os.Interrupt` is SIGINT on posix.

I left SIGHUP out as that is typically reserved for hot reloading, rather than terminating the program. It might be worth adding that at a later date.

Is SIGQUIT something we really want to be catching?

Reply:
I'm ok with SIGINT and SIGTERM only.